src/store/services/getters.ts

```diff
@@ -6,14 +8,14 @@ import { State as RootState, RootStore } from '../state';
 const { read } = getStoreAccessors<ServicesState, RootState>('services');
 
 const getters = {
-  servicesById: (state: ServicesState): { [id: string]: Service } => state.byId,
+  servicesById: (state: ServicesState): { [id: string]: Service } => state.services,
 
   serviceIds(state: ServicesState): string[] {
-    return state.allIds;
+    return map(state.services, (_: Service, key: string) => key);
```

Comment:
If you want the keys of an object as a list of strings in JavaScript you can use `Object.keys(state.services)`